.github/workflows/docker.yml

```diff
@@ -94,7 +94,9 @@ jobs:
           context: .
           file: ${{ env.DOCKERFILE }}
           platforms: ${{ env.DOCKER_PLATFORMS }}
-          push: ${{ github.event_name != 'pull_request' && github.actor != 'dependabot[bot]' }}
+          push: ${{ (github.event_name != 'pull_request' &&
+                     github.actor != 'dependabot[bot]') ||
+                     startsWith(github.ref, 'lychee-v') }}
```

Comment:
I think we always want to push -- even when we commit to `master`. Otherwise, the Docker image cannot be tested before release.